src/node_api_jsrt.cc

```diff
@@ -176,6 +217,9 @@ namespace jsrtimpl {
       jsrtimpl::ExternalCallback* externalCallback =
         reinterpret_cast<jsrtimpl::ExternalCallback*>(callbackState);
 
+      // Make sure any errors encountered last time we were in N-API are gone.
```

Comment:
Is this the behavior in v8 too? I'm not sure if we want the previous error to be silently swallowed here